### DISCUSSION TITLE
MacBook Pro screen brightens automatically

### DISCUSSION THREAD

**Discussion by artgoldberg** [MonitorControl/MonitorControl#997] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

MonitorControl brightness controls MacBook Pro screen, but it automatically gets brighter and too bright soon after.
Same problem occurs with the brightness control on the Touch Bar.

### Expected behavior

Brightness of MacBook Pro screen stays at level I select. 

### Anything else?

I use f.lux, but cannot find "Avoid gamma table manipulation" under Preferences » Displays. F.lux is current version 41.4.

### Environment Information (please complete the following information)

```markdown
- macOS version: 11.6
- Mac model: MacBook Pro (13-inch 2020)
- MonitorControl version: 4.0.2
- Monitor(s): 2 ViewSonic VA2055Sm monitors
- Monitor Cable(s)/Connection(s): DVI  - USB 3.1 and VGA - StarTech MDP2VGA - 4K UHD - USB 3.1
- Apple Silicon/M1 (yes or no): no, Intel Core i7
```

**Comment by the0neyouseek** [comment]
Hi @artgoldberg

Your brightness may be changing because of the system option "Automatically adjust brightness" that adjust brightness using the value from your macbook ambient light sensor.

You can disable this options following this guide https://support.apple.com/guide/mac-help/change-your-displays-brightness-mchlp2704/mac

Also, here's where the "Avoid gamma table manipulation" option is located in app settings.
<img width="412" alt="pref_4" src="https://user-images.githubusercontent.com/4009175/159115127-02ff1739-fc7c-4f04-a889-3e79d789e5a0.png">

Tell me if fixes you issue so I can close this.
Have a nice day.

**Comment by jmreeves11** [comment]
Is there a best practice for brightness settings across devices and applications when they are all working together or at least plugged in to each other? 

Should I keep my MacBook Pro’s native setting for automatic brightness adjustment “on” while also toggling “on” the MonitorControl “avoid gamma adjustment” setting? What would be the result of that? Which screen(s)/display(s) would adjust automatically and which would not?

What happens if I do use f.lux? 
What happens if my Mac auto brightness setting is on?
If I keep either of those (f.lux or native auto adjuster) on, what is the issue for Monitor control? If I keep them on and check the disable gamma button in MonitorControl settings, what will the effect be on my user experience with my displays?


Thanks for all the clarification. 
Sincerely,
A non-technically savvy person
